sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -255,3 +287,43 @@ case class AlterViewAsCommand(
     session.sessionState.catalog.alterTable(updatedViewMeta)
   }
 }
+
+/**
+ * Drops a view which matches the given name.
+ *
+ * If the view name contains database prefix, this command will drop a permanent view matching the
+ * given name, or throw an exception if view not exist. Else, this command will try to drop a local
+ * temporary view first, if view not exist, try global temporary view next, if still not exist, try
```

Comment:
I think it is a bit risky to provide user the ability to drop a global view implicitly. 

I think all global view modification commands should be done explicitly, including create, drop, alter..